Soomla/Assets/Plugins/Soomla/Profile/SoomlaProfile.cs

```diff
@@ -120,10 +132,6 @@ public class SoomlaProfile
                     }
                 }
             };
-
-            #if UNITY_EDITOR
-            TryFireProfileInitialized();
-            #endif
```

Comment:
@borisspektor, there is: https://github.com/vedi/unity3d-profile/blob/muticall-initialize-callbak/Soomla/Assets/Plugins/Soomla/Profile/SoomlaProfile.cs#L886